### DISCUSSION TITLE
Automatically run inlay-hints

### DISCUSSION THREAD

**Discussion by InsigMath** [LazyVim/LazyVim#1219] [Q&A]
Hello,

I hope this is not a silly question, I know how to turn on inlay-hints for rust/c/c++ but I would like it so that inlay hints are enabled automatically (or at least if it is a file format that has such a feature). How do I make inlay-hint on automatically? Thank you!

**Comment by dpetka2001** [comment]
```lua
{
  "neovim/nvim-lspconfig",
  opts = {
    inlay_hints = {
      enabled = true,
    },
  },
}
```
If you put this in your spec I think you should get what you want.

**Reply by jyuan0** [reply]
And just to avoid any confusion, the code above requires Neovim >= 0.10.0. Otherwise, try: https://github.com/lvimuser/lsp-inlayhints.nvim
